modules/server/src/main/scala/navigate/server/tcs/TcsEpicsSystem.scala

```diff
@@ -993,6 +1031,30 @@ object TcsEpicsSystem {
       )
     override val oiWfsCmds: WfsCommandsChannels[F]                                       =
       WfsCommandsChannels.build(channels.telltale, channels.oiwfs)
+
+    override val p1GuiderGainsCmd: Command3Channels[F, Double, Double, Double] =
+      Command3Channels(
+        channels.telltale,
```

Comment:
PWFS1 channels come from a different IOC, they should use a different telltale channel. Same for PWFS2 and OIWFS.
For PWFS1: pwfs1:health.VAL
for PWFS2: pwfs2.health.VAL
for OIWFS: ag:hlth:oiwfs:health.VAL